types/react-with-styles/index.d.ts

```diff
@@ -4,13 +4,14 @@
 //                 Brie Bunge <https://github.com/brieb>
 //                 Joe Lencioni <https://github.com/lencioni>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.2
 
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import { CSSProperties } from 'aphrodite';
 
-interface Theme { [key: string]: any; }
```

Comment:
I'm not sure I understand what this enables. Can you give me an example of what you couldn't do before and after this change? Also, will this cause errors for anyone and make it difficult to update?

Reply:
Removing the index signature makes it possible to extend this interface with type-safe themes. There is an example in the test. With this index signature any other property from theme such as `unitt` would also work.

Reply:
with regarding to upgrading:

if users didn't have a type-safe theme definition they can add the index signature themselves:

```ts
declare module 'react-with-styles' {
  interface Theme {
    [key: string]: any;
  }
}
```

Reply:
Ah I see now. Thank you for explaining.